utils/shellcompletion.go

```diff
@@ -7,62 +7,71 @@ import (
 	"io/ioutil"
 	"net/http"
 	"os"
+
+	"github.com/urfave/cli"
 )
 
-var appName = "synse" // TODO: Dynamically source this from app.Name
 var autocompletionPath = "/etc/bash_completion.d/"
```

Comment:
What if someone wants a different path? I think this path might not also be the default one for some `*nix` (and possibly bash on windows, although fuck that shit) systems. Also might cause problems for `zsh` completion.

Reply:
good call - created #104 to track this